src/main/java/com/rakovets/course/java/core/practice/exception_handling/Task6.java

```diff
@@ -0,0 +1,22 @@
+package com.rakovets.course.java.core.practice.exception_handling;
+
+import com.rakovets.course.java.core.practice.exception_handling.exception.FirstException;
+import com.rakovets.course.java.core.practice.exception_handling.exception.SecondException;
+import com.rakovets.course.java.core.practice.exception_handling.exception.ThirdException;
+
+import java.io.FileNotFoundException;
+import java.util.Random;
+
+public class Task6 {
+
+    public void generateException() throws FirstException, SecondException, ThirdException {
+        int random = (int) (Math.random()*3+1);
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"